src/Illuminate/Foundation/Testing/AssertionsTrait.php

```diff
@@ -14,7 +14,7 @@ public function assertResponseOk()
 	{
 		$actual = $this->response->getStatusCode();
 
-		return PHPUnit::assertTrue($this->response->isOk(), 'Expected status code 200, got ' .$actual);
+		return PHPUnit::assertTrue($this->response->isOk(), "Expected status code 200, got $actual");
```

Comment:
Thought about that, but the original code didn't have that. And full stop (the dot) is a character by itself: in a context where you compare the expected with the actual, I think introducing additional chars would be a bad idea.